### ISSUE TITLE
Payment Options In Barion web payment page

### ISSUE THREAD

**Issue by mukhammadalime** [aron123/node-barion#68]
Hello. This is not an issue. I am just wondering if It is possible to remove all other payment options except the barion wallet in the Responsive Barion Web Payment page. If you help me, I would definitely be grateful. Thanks in advance.
![Screen Shot 2022-09-23 at 1 30 00 AM](https://user-images.githubusercontent.com/98612897/191806853-d5df04c9-1986-4da3-aae3-03a5a9625521.png)

**Comment by aron123** [comment]
Yes, you can a start a payment with `FundingSources` set to `["Balance"]`. Also, this option can be set as a default value when instantiating a `new Barion()` object.

**Comment by mukhammadalime** [comment]
I really appreciate that you replied in minutes.
I put only "Balance" in the fundingsources array and try to make a payment, but it still shows me other options too. Does it still show but can not be paid when I put only "Balance" or shouldn't it show other options? Can you please help me with this too?

**Comment by mukhammadalime** [comment]
Thank you so much. I really appreciate your help!

**Comment by aron123** [comment]
I think that it shouldn't show other options, but I can only test it tomorrow afternoon at the earliest. Maybe you can try to set `GuestCheckOut` to `false` too.

**Comment by mukhammadalime** [comment]
I forgot to put "Balance" in orderPayment object that will be passed into barion.startpayment(). After I put it there too, It only showed me the barion wallet. Thank you so much.

**Comment by mukhammadalime** [comment]
You also deleted your comment after I deleted mine😅.

**Comment by aron123** [comment]
Glad to hear that it works now. Feel free to reopen this issue if you have any further questions.
